rust/operator-binary/src/kafka_controller.rs

```diff
@@ -420,6 +452,16 @@ pub async fn reconcile_kafka(kafka: Arc<KafkaCluster>, ctx: Arc<Ctx>) -> Result<
         .await
         .context(FailedToInitializeSecurityContextSnafu)?;
 
+    tracing::debug!("Security settings: kerberos enabled/secret-class: {}/{:#?} tls enabled/client-auth-class: {}/{:#?}",
```

Comment:
One nice thing about tracing is that you can capture variables in a structured way, you don't need to squash everything into a string. I  can recommend https://docs.rs/tracing/latest/tracing/#recording-fields
```suggestion
    tracing::debug!(
        kerberos_enabled = kafka_security.has_kerberos_enabled(),
        kerberos_secret_class = ?kafka_security.kerberos_secret_class(),
        tls_enabled = kafka_security.tls_enabled(),
        tls_client_authentication_class = ?kafka_security.tls_client_authentication_class(),
        "The following security settings are used"
    );
```

Reply:
Thanks! Will take a look at that